packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/common.ts

```diff
@@ -58,7 +76,104 @@ export interface CommonDestinationProps {
    * @default - if `logging` is set to `true`, a log group will be created for you.
    */
   readonly logGroup?: logs.ILogGroup;
+}
 
+/**
+ * Common properties for defining a backup, intermediary, or final S3 destination for a Kinesis Data Firehose delivery stream.
+ */
+export interface CommonS3Props {
```

Comment:
Should this be called `CommonDestinationS3Props`? Similarly how it's `DestinationLogggingProps`?

Reply:
I still think this one is a bit confusing. But it is not customer-facing, so I will change it. I think it's better than `CommonS3Props`

Reply:
Hmmm, why do you say this is not customer-facing?